### PR TITLE
Fix Log bug

### PR DESCRIPTION
**Pull request by wang-chen**
`torch.sign(x)` can return `0` when `x` is zero, which is not expected.

### CODE REVIEW

**Comment by wang-chen** [comment]
@NicolayP this PR should solve your problem.
Will merge it after a full test, and then release it once merged.

**Review by xukuanHIT — COMMENTED** [review]
OK

File: pypose/lietensor/lietensor.py
Comment (wang-chen) [inline pypose/lietensor/lietensor.py:1111]: @haleqiu  please have a look at this change. Could you merge this PR, if it meets your expectation?

**Review by haleqiu — APPROVED** [review]

**Pull request merged**
